libs/openFrameworksCompiled/project/osx/config.osx.default.mk

```diff
@@ -78,7 +78,7 @@ ifndef MAC_OS_STD_LIB
 endif
 
 ifndef MAC_OS_CPP_VER
-    MAC_OS_CPP_VER = -std=c++17
+    MAC_OS_CPP_VER = -std=c++20
```

Comment:
@dimitre was this meant to slip in? 🙂

Reply:
oh no! 😓I'll be reverting now in a new PR

Reply:
done and pushed to master. good catch @ofTheo thanks